db/migrate/20210223152748_add_operator_to_relative_date_filters.rb

```diff
@@ -0,0 +1,43 @@
+class AddOperatorToRelativeDateFilters < ActiveRecord::Migration[6.1]
+  def up
+    UserFilter.where('contents LIKE ?', '%relativeOption%').find_each do |user_filter|
+      filters = JSON.parse(user_filter.contents)
+      filters.each { |filter| add_operator_and_has_timestamp_to_filter(filter) }
+      filters.reject! { |filter| filter.dig('filterOption', 'hasTimestamp') && filter['value'].class == Hash && filter['value']['when'] == 'next' }
```

Comment:
Wasn't the decision that we didn't want to remove these saved user filters completely but rather adjust them such that they didn't have this condition?

Reply:
The decision was made to remove the filter option incase there were other filter options (not just the relative date one).  Might be worth updating the migration to remove all of the saved filters if that is the only filter option in them.

Decision was documented here: https://mitrecorp.slack.com/archives/C01A6P94R9B/p1614174233038600

Reply:
Right that's what I'm saying - it didn't seem like this migration was doing that at first, but at a second look I think it is, so disregard that.

>Might be worth updating the migration to remove all of the saved filters if that is the only filter option in them.

That would make sense to me as well, good point.

Reply:
That's what I suggested in the slack thread, but it was not confirmed by Kelly or anyone else to remove the entire preset